app/views/admin/instances/show.html.haml

```diff
@@ -124,6 +124,22 @@
 
   %hr.spacer/
 
+  %h3#instance-notes= t 'admin.instances.notes.title'
+  %p= t 'admin.instances.notes.description_html'
+  .report-notes
+    = render partial: 'admin/report_notes/report_note', collection: @instance_notes
+
+  = simple_form_for @instance_note, url: admin_instance_notes_path(instance_id: @instance.domain) do |form|
```

Comment:
I think this form probably needs to be conditional on "can create instance notes" (I forget the syntax for that in haml)

Reply:
This isn't necessary for now as the permission of "can create instance notes" is the same as the permission for "can view instances" — both need `can(:manage_federation)` for now. Perhaps in the future we could allow the show action and allow creating an instance note if you can manage reports?